apps/src/applab/designElements/textInput.jsx

```diff
@@ -80,15 +83,13 @@ class TextInputProperties extends React.Component {
           handleChange={this.props.handleChange.bind(this, 'fontFamily')}
         />
         <PropertyRow
-          desc={'font size (px)'}
+          desc={applabMsg.designElementProperty_fontSizePx()}
           isNumber
           initialValue={parseInt(element.style.fontSize, 10)}
           handleChange={this.props.handleChange.bind(this, 'fontSize')}
         />
-        <EnumPropertyRow
```

Comment:
If I were making this change in the future, I would separate the creation of the TextAlignmentPropertyRow component from the rest of the i18n changes since they're distinct updates.

Reply:
I understand, but in this case, it doesn't seem like a big enough change to make it a separate task.
We don't change the business logic, we just move some code to a separate component.

Reply:
The difference is how you would test each of the changes. The refactor change is tested via the tests you added and visual checking that they're the same before and after. The i18n changes add new keys that can be translated, testing that can be done via adding te-ST translations and checking to see what changes visually with the new translations.

Reply:
We don't need to and we shouldn't test the I18n lib functionality for every component.
If we use the I18n functions and it displays English text without breaking anything it means it works.

Reply:
I'm relatively new, but it seems like my teammates do tend to appreciate PRs that are smaller in scope to make the reviewing process more manageable as well.